aws_doc_sdk_examples_tools/lliam/test/services_test.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from typing import List
+
+from aws_doc_sdk_examples_tools.lliam.domain.commands import CreatePrompts
+from aws_doc_sdk_examples_tools.lliam.domain.model import Prompt
+from aws_doc_sdk_examples_tools.lliam.service_layer.create_prompts import create_prompts
+from aws_doc_sdk_examples_tools.lliam.service_layer.unit_of_work import AbstractUnitOfWork
+from aws_doc_sdk_examples_tools.lliam.adapters.repository import (
+    AbstractPromptRepository,
+    AbstractDocGenRepository,
+)
+
+
+class FakePromptRepository(AbstractPromptRepository):
```

Comment:
Too many fakes when you could just use a single Fs fake

Reply:
Fixed